packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -161,40 +165,69 @@ export const MafsGraph = (props: MafsGraphProps) => {
                         </Mafs>
                     </View>
                 </View>
-                {renderGraphControls({state, dispatch})}
+                {renderGraphControls({state, dispatch, width})}
             </View>
         </GraphConfigContext.Provider>
     );
 };
 
 const renderPointGraphControls = (props: {
-    state: InteractiveGraphState;
+    state: PointGraphState;
     dispatch: (action: InteractiveGraphAction) => unknown;
+    width: number;
 }) => (
-    <Button
-        kind="secondary"
+    <View
         style={{
-            width: "100%",
-            marginLeft: "20px",
-        }}
-        onClick={() => {
-            props.dispatch(actions.pointGraph.addPoint([0, 0]));
+            flexDirection: "row",
+            width: props.width,
         }}
     >
-        Add Point
-    </Button>
+        <Button
+            kind="secondary"
+            style={{
+                width: "100%",
+                marginLeft: "20px",
+            }}
+            onClick={() => {
+                props.dispatch(actions.pointGraph.addPoint([0, 0]));
+            }}
+        >
+            Add Point
+        </Button>
+        {(props.state.focusedPointIndex !== null ||
+            props.state.previouslyFocusedPointIndex !== null) && (
+            <Button
+                kind="secondary"
+                color="destructive"
+                style={{
+                    width: "100%",
+                    marginLeft: "20px",
+                }}
+                onClick={(event) => {
+                    props.dispatch(
+                        actions.pointGraph.removePoint(
+                            props.state.previouslyFocusedPointIndex!,
```

Comment:
I don't quite understand how this works. We are checking `props.state.focusedPointIndex !== null || props.state.previouslyFocusedPointIndex !== null` to decide whether to show the button, but the `previouslyFocusedPointIndex` is the one that's deleted. Should we omit the check for `props.state.focusedPointIndex !== null`?

Reply:
previouslyFocusedPointIndex gets cleared when the point is removed (it's in the reducer).

We need this because the button steals the focus and the point blurs and then we're not able to remove the point. I wish it was different :( I'd be happy to continue to iterate in a follow on (and pair on it!) if you'd like

Reply:
Yeah, I get why we need both `previouslyFocusedPointIndex` and `focusedPointIndex`. What I don't understand is why we always delete `previouslyFocusedPointIndex` here. Based on your explanation, I would expect logic like this:

```ts
actions.pointGraph.removePoint(
  // delete the focused point, or the previously focused point if no point is focused.
  props.state.focusedPointIndex ?? props.state.previouslyFocusedPointIndex,
)
```

I'm running your code locally, and the currently focused point _does_ get deleted when I click the button, as I would expect. So it's working. I just don't understand _why_ it works.

Reply:
Ohhh, I get it... the point is _always_ blurred, because the button gets focused when it's clicked. Okay. That might be worth a comment :)